internal/service/federateddatabaseinstance/data_source_federated_database_instance_test.go

```diff
@@ -47,31 +47,30 @@ func TestAccDataSourceFederatedDatabaseInstance_basic(t *testing.T) {
 	})
 }
 
-func TestAccDataSourceFederatedDatabaseInstance_S3Bucket(t *testing.T) {
-	acc.SkipTestExtCred(t)
+func TestAccFederatedDatabaseInstanceDS_s3Bucket(t *testing.T) {
 	var (
 		resourceName      = "data.mongodbatlas_federated_database_instance.test"
 		orgID             = os.Getenv("MONGODB_ATLAS_ORG_ID")
 		projectName       = acctest.RandomWithPrefix("test-acc")
 		name              = acctest.RandomWithPrefix("test-acc")
 		policyName        = acctest.RandomWithPrefix("test-acc")
-		roleName          = acctest.RandomWithPrefix("test-acc")
+		roleName          = acctest.RandomWithPrefix("mongodb-atlas-test-acc")
```

Comment:
[nit] adding `federated` help to understand that the role is used for federated tests (this can be applied to other var name as well)
```suggestion
		roleName          = acctest.RandomWithPrefix("mongodb-atlas-federated-test-acc")
```

Reply:
if that's fine I've changed to mongodb-atlas-test-acc-fed